src/opcua-iiot-result-filter.ts

```diff
@@ -158,7 +163,22 @@ module.exports = (RED: nodered.NodeAPI) => {
       return result
     }
 
-    const convertResult = (msg: FilterInputPayload, result: Todo) => {
+    const convertAllResults = (payload: FilterInputPayload, result: Todo | Todo[]) => {
+      if (!isArray(result)) {
+        return convertResult(payload, result)
+      } else {
+        return result.map((item: Todo) => {
```

Comment:
I would change

```ts
    return result.map((item: Todo) => {
```

to 

```ts
    return result.map((item) => {
```

because TypeScript should be able to infer the type from `result` once you have a given it a type right? Worst case:
You later give `result` a type and forget the `item: Todo`. Then you could run into errors, because you are removing the type information that typescript would otherwise be able to infer.

To illustrate what i mean:

```ts
declare const foo: string[]
foo.map((item: any) => item.thisDoesNotExist()) 
// TypeScript is fine with this
```